tests/parser/parser.cpp

```diff
@@ -1605,7 +1613,6 @@ TEST(Parser, parseIdsOnEverything)
     EXPECT_EQ("u3id", model->units("units3")->id());
     EXPECT_EQ("v1id", model->component("component2")->variable("variable1")->id());
     EXPECT_EQ("r1id", model->component("component2")->reset(0)->id());
```

Comment:
We are not testing the `id()` of `variable2` and of the `reset`, `test_value` and `reset_value` elements. (However, I guess it's "tricky" to test the `id()` of the math elements.)

Reply:
There's no rationale to "test" an id anyway?  They can be anything, including absent, so I'm not sure what would you actually be testing?

Reply:
I would indeed only test that it works on any `Entity` object. It's just that the test reads `parseIdsOnEverything`, so I am reviewing in that context...

Reply:
Fairynuff :)

Reply:
I might change the test title to 'parseIdsOnEverythingButMath'.

Reply:
Since the `id()` of `variable1` is tested we infer that the `id()` of `variable2` will also be found.

Reply:
Am I correct in thinking that nothing has been done about this? (Just checking.)

Reply:
Scrap my last comment. Argh!